roofit/xroofit/src/xRooNLLVar.cxx

```diff
@@ -323,7 +393,8 @@ void xRooNLLVar::reinitialize()
       std::set<std::string> attribs;
       if (std::shared_ptr<RooAbsReal>::get())
          attribs = std::shared_ptr<RooAbsReal>::get()->attributes();
-      this->reset(std::unique_ptr<RooAbsReal>{fPdf->createNLL(*fData, *fOpts)}.release());
+      this->reset(fPdf->createNLL(*fData, *fOpts));
```

Comment:
```c++
this->reset(std::unique_ptr<RooAbsReal>{fPdf->createNLL(*fData, *fOpts)}.release());
```
Also here, the memory safe interface mode requires us to first wrap the NLL in a `std::unique_ptr`, so it can be constructed from both the raw pointer or `unique_ptr`.